pallets/stateful-storage/src/test_common.rs

```diff
@@ -28,3 +28,61 @@ pub mod constants {
 	pub const PAGINATED_APPEND_ONLY_SCHEMA: SchemaId = 107;
 	pub const UNDELEGATED_PAGINATED_SCHEMA: SchemaId = 108;
 }
+
+#[cfg(test)]
+pub mod test_constants {
```

Comment:
this is not a constant module more like a test utility module

Reply:
Changed to `test_utility`